fip-0023.md

```diff
@@ -0,0 +1,46 @@
+---
+fip: X
+title: Temporary adjustment of reserves to pay Block Producers
+status: Draft
+type: Functionality
+author: Sukesh Kumar Tedla <contact@greeneosio.com>
+created: 2020-12-24
+updated: 
+---
+
+# Abstract
+As described in the knowledge base, the genesis of the FIO chain included 10M tokens set aside for Block Producer Reserves, to be minted each day as needed if the normal fee distribution falls below 50k FIO tokens. In December 2020, these reserves were exhausted and the current BP pay has dropped significantly. During the recent BP call, it was discussed that the pay from organic on-chain activities is about 10% of what it was with the BP reserves. The reduced pay has a direct impact on BPs operating efficiently which is essential for the reilience of the FIO network. To address this issue, FIO Foundation and BPs have been discussing multiple proposals focused on short-term and long-term scenarios.
+This proposal is aimed at implementing a short-term solution to address the issue now and to secure some more time to work on a permanent solution.
+
+* For more detailed discusson on this topic, please visit: https://fioprotocol.atlassian.net/wiki/spaces/FC/pages/49184971/The+Future+of+Block+Producer+Reserves 
+
+Proposed new actions:
+Contract|Action|Description|
+|---|---|---|
+|fio.treasury|Updating BPMAXRESERVE|Need to update the BPMAXRESERVE value from 10000000000000000(10M Tokens) to 15000000000000000(15M Tokens) |
+
+# Motivation
```

Comment:
Can you add some details here? Looks like it's currently just the template. Maybe show some info on what BP costs are and how the current fee-based approach without any subsidization from the foundation at this time isn't quite enough to keep BPs engaged and working on FIO.

Reply:
I added the details now.